### ISSUE TITLE
[FEATURE REQUEST] MIDI CC-to-CV - Other method than hold right mouse button

### ISSUE THREAD

**Issue by DomiKamu** [VCVRack/Rack#563]
OS (if applicable): Windows (all versions)
Version (or "dev" if compiling from source): 0.5.1

Hi, is it possible to rework a bit the "MIDI learn" feature for MIDI CC-to-CV module, because holding right mouse button while using controller to do a MIDI learn isn't user friendly, in **particular when the MIDI controller is far from (wired) mouse.** Thanks a lot in advance.

**Comment by DomiKamu** [comment]
...the best way in my opinion is to do the MIDI learn while the field is in edit mode (after left click, when you can type CC number).

**Comment by AndrewBelt** [comment]
Yeah, that makes sense. A MIDI CC signal would set and defocus the input.

**Comment by DomiKamu** [comment]
For sure Andrew, it will be great! Thanks a million for VCV Rack, it's my new game (now Kerbal Space Program is jealous lol) great stuff!

**Comment by LKHSogpit** [comment]
There is a midi learn function in v0.5.1? How do you do it?
